gateway/1.5/modules/ROOT/pages/_partials/kubernetes-service-note.adoc

```diff
@@ -0,0 +1,10 @@
+If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, you must set the following parameters in the `service` resource:
```

Comment:
```suggestion
If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, set the following parameters in the `service` resource:
```
If the modal is not required, please omit it. See [modal verbs](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=modal%20verbs.-,modal%20verbs,-Although%20some%20specific)

Reply:
OK! Thanks!